fl4health/clients/moon_client.py

```diff
@@ -23,10 +27,13 @@ def __init__(
         metrics: Sequence[Metric],
         device: torch.device,
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
-        checkpointer: Optional[TorchCheckpointer] = None,
         temperature: float = 0.5,
-        contrastive_weight: Optional[float] = None,
         len_old_models_buffer: int = 1,
+        beta_update_interval: int = 20,
+        checkpointer: Optional[TorchCheckpointer] = None,
+        contrastive_weight: Optional[float] = None,
+        mkmmd_loss_weights: Optional[Tuple[float, float]] = None,
+        feature_l2_norm: Optional[float] = None,
```

Comment:
Very minor, but I would suggest renaming this to include `_weight` at the end to be a bit clearer that this isn't actually a "norm"